src/App.tsx

```diff
@@ -1,24 +1,186 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { SortBy } from './types/SortBy';
+import { addTodo, deleteTodo } from './api/todos';
+import { Todos } from './components/Todos';
+import { useGetTodos } from './hooks';
+import { Errors } from './types';
 
-const USER_ID = 0;
+const USER_ID = 11361;
 
 export const App: React.FC = () => {
+  const [sortBy, setSortBy] = useState<SortBy>(SortBy.all);
+  const [selectedTodo, setSelectedTodo] = useState<number[]>([]);
+  const [inputValue, setInputValue] = useState<string>('');
+  const [isLoadingTodo, setIsLoadingTodo] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const {
+    isLoading,
+    todos,
+    errorMessage,
+    todosNotCompleted,
+    handleError,
+    handleIsTodoDeleted,
+  } = useGetTodos(USER_ID, tempTodo);
+
+  const deleteOneTodo = async (todoId: number) => {
+    try {
+      await deleteTodo(USER_ID, todoId);
+    } catch (error) {
+      handleError(Errors.delete);
+    }
+
+    handleIsTodoDeleted(true);
+    setSelectedTodo(prevSelectedTodo => [...prevSelectedTodo, todoId]);
+  };
+
+  const handleDeleteTodo = (value: number) => deleteOneTodo(value);
+
+  const handleKeyDown = async (
```

Comment:
This can be called on form submit rather than on pressing enter in the input
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/87986955/ed69942e-1ff7-488e-b0f0-0d1a579198cd)
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/87986955/ef1b8c7d-bd75-456f-b670-4d70e83fd48e)